src/components/interaction-designer/ResourceViewer.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+  <div class="resource-viewer">
+    <div class="resource-viewer-contents">
+      <resource-viewer-block
+        v-for="block in activeFlow.blocks"
+        :id="`block/${block.uuid}`"
+        :key="block.uuid"
+        :block="block"
+        @change="debounce_persistFlow" />
+    </div>
+  </div>
+</template>
+
+<script lang="ts">
+import {mixins} from 'vue-class-component'
+import {Component} from 'vue-property-decorator'
+import {namespace} from 'vuex-class'
+import Lang from '@/lib/filters/lang'
+import {IContext, IFlow} from '@floip/flow-runner'
+import {debounce} from 'lodash'
+
+const flowVuexNamespace = namespace('flow')
+const builderVuexNamespace = namespace('builder')
+
+export const DEBOUNCE_FLOW_PERSIST_MS = 1500
+
+@Component({})
+export class ResourceViewer extends mixins(Lang) {
+  @flowVuexNamespace.Getter activeFlow!: IFlow
+
+  mounted(): void {
+    console.debug('VueJS flow resources viewer mounted!')
+  }
+
+  get id(): string {
+    return this.$route.params.id
+  }
+
+  debounce_persistFlow = debounce(this.persistFlowAndHandleUiState.bind(this), DEBOUNCE_FLOW_PERSIST_MS)
+
+  // this should go after debounce_persistFlow
+  @builderVuexNamespace.Action persistFlowAndHandleUiState!: () => Promise<IContext | undefined>
```

Comment:
@bulatgab , for some reason I got this when running `yarn build-bundle`
<img width="1317" alt="Screen Shot 2022-10-07 at 6 07 32 PM" src="https://user-images.githubusercontent.com/68745918/194678005-571ab8b9-70c5-4a29-9c56-be8cc3b4206b.png">

it's not affecting the voto5 side though, but I'm wondering if we can try to fix it now